mkdocs.yml

```diff
@@ -168,8 +169,6 @@ markdown_extensions:
   - pymdownx.superfences
   - pymdownx.highlight
   - admonition
-  - markdown_include.include:
```

Comment:
We started getting an error
```
INFO     -  Building documentation...
ERROR    -  Config value: 'markdown_extensions'. Error: MarkdownInclude.extendMarkdown()
            missing 1 required positional argument: 'md_globals'
Aborted with 1 Configuration Errors!
```
It might be some outdated/incompatible package version, but while investigating it we found that we use that extension no more.